weasyl/moderation.py

```diff
@@ -608,8 +607,6 @@ def editcatchphrase(userid, otherid, content):
 
 
 def bulk_edit(userid, action, submissions=(), characters=(), journals=()):
-    web.header('Content-Type', 'text/plain')
```

Comment:
Why would you? It doesn’t seem right for this action to deal with response headers.